### ISSUE TITLE
Add ability to change camera near clipping

### ISSUE THREAD

**Issue by brianconnoly** [hrydgard/ppsspp#19114]
### What should happen

PPSSPP should have setting to adjust camera near clipping distance

Playing PPSSPP VR, and moving camera to character head breaks all immersion because object disappear in front of you (

### Who would this benefit

All PPSSPP VR players that want to play in first person camera

GTA Stories is great example of such game
PPSSPP gives great stereo 360 image, and camera control, but lack of control over near clipping destroys possibility to play in first repson

### Platform (if relevant)

Android

### Games this would be useful in

ULUS-10041 Grand Theft Auto: Liberty City Stories

### Other emulators or software with a similar feature

Every game engine has capability to change near/far distance
Even some games has that setting (90s / early 00s)

### Checklist

- [X] Check the [latest git build](https://www.ppsspp.org/devbuilds) in case it's already implemented.
- [X] [Search for other requests](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same feature.

**Comment by sum2012** [comment]
@lvonasek

**Comment by lvonasek** [comment]
I will look into it during June.

**Comment by lvonasek** [comment]
I did a quick test and when I set a near clipping plane using projection matrix then it does nothing in GTA.

**Comment by brianconnoly** [comment]
Did you zoomed close enough?

**Comment by lvonasek** [comment]
Yes. The near clip plane is hardcoded in the code: https://github.com/hrydgard/ppsspp/blob/cc5763cffdb60488785e41bf2460e3df09d27a45/Common/VR/PPSSPPVR.cpp#L649

When I modified it, it had a zero effect on the games I tried.
